int8[] robotIDs
Belief fix_paths
Pose2DList current
Pose2DList target
---
Belief path
